src/routes/index.js

```diff
@@ -2,29 +2,37 @@ import { BrowserRouter, Navigate, Route, Routes } from 'react-router-dom';
 
 import Dashboard from '../pages/dashboard';
 import Nitro from '../pages/dashboard/nitro';
-import Loading from '../pages/loading';
+import LoadingScreen from '../common/components/LoadingScreen';
 import Login from '../pages/login';
 import Register from '../pages/register';
 import FriendsRouters from './routes-friends';
 
 function AppRoutes() {
   return (
-    <BrowserRouter>    
-        <Routes>
-          <Route path="/*">
-            <Route index element={<Loading />} />
-            <Route path="login/" element={<Login />} />
-            <Route path="register/" element={<Register />} />
-            <Route path="dashboard/*">
-              <Route index element={<Dashboard />} />
-              <Route path="friends/*">
-                <Route index element={<Navigate replace to="online" />} />
-                <Route path="*" element={<FriendsRouters />} />
-              </Route>
-              <Route path="nitro/" element={<Nitro />} />
+    <BrowserRouter>
+      <Routes>
+        <Route path="/*">
+          <Route
+            index
+            element={
+              <LoadingScreen
+                title="DID YOU KNOW"
+                subtitle="We’re glad you’re back!"
+              />
+            }
+          />
```

Comment:
We don't need to use a loading screen as part of the routing system. It is used when loading some data, so we need to rely on loading state but not on route path